docker-sync.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+options:
+  verbose: true
+  compose-dev-file-path: 'docker-compose-sync.yml'
+syncs:
+  app-sync:
+    src: './'
+    sync_strategy: 'rsync'
```

Comment:
Haven't entirely settled on a sync strategy yet, but `rsync` worked better for me than `native_osx`.